emulator/clients/interrupter.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
```

Comment:
A little description of this client would be good

Reply:
I added the two descriptions in the latest commit